app/index.html

```diff
@@ -130,33 +130,36 @@ <h1 class="brand-logo center">Cartograms 4 All</h1>
         </div>
 
         <!-- Website Footer -->
-        <footer class="page-footer">
-            <div class="container">
-                <div class="row">
-                    <div class="col l6 s12">
-                        <h5 class="white-text">Cartograms 4 All</h5>
-                        <p class="grey-text text-lighten-4">A group project for Professor Jullig's UCSC CMPS 115 Summer 2017</p>
-                    </div>
-                    <div class="col l4 offset-l2 s12">
-                        <h5 class="white-text">Team</h5>
-                        <ul>
-                            <li><a class="grey-text text-lighten-3" href="#!">Casey Hillers</a></li>
-                            <li><a class="grey-text text-lighten-3" href="#!">Luke Tanner</a></li>
-                            <li><a class="grey-text text-lighten-3" href="#!">Jiayao Lin (Kevin)</a></li>
-                            <li><a class="grey-text text-lighten-3" href="#!">Jeffrey Larocque</a></li>
-                            <li><a class="grey-text text-lighten-3" href="#!">Michael Crane</a></li>
-                            <li><a class="grey-text text-lighten-3" href="#!">Ahmed Almutawa</a></li>
-                        </ul>
-                    </div>
-                </div>
-            </div>
-            <div class="footer-copyright">
-                <div class="container center text-lighten-3">
-                    Based off of code provided by <a href="https://github.com/trburkhart/Cartowebsite">Tim Burkhart</a>
-                </div>
-            </div>
-        </footer>
-    </div>
+     <footer class="page-footer">
+      <div class="container">
+        <div class="row">
+          <div class="col l6 s12">
+            <h5 class="white-text">Cartograms 4 All</h5>
+            <p class="grey-text text-lighten-4">Cartograms4All is a simple application
+            that allows user to create useful visualization and fully explore the power of their own data    
+            </p>
+            <p class="grey-text text-lighten-4">A group project for Professor Jullig's UCSC CMPS 115 Summer 2017</p>
+          </div>
+          <div class="col l4 offset-l2 s12">
+            <h5 class="white-text">Team</h5>
+            <ul>
+              <li><a class="grey-text text-lighten-3" href="#!">Casey Hillers</a></li>
+              <li><a class="grey-text text-lighten-3" href="#!">Luke Tanner</a></li>
+              <li><a class="grey-text text-lighten-3" href="#!">Jiayao Lin (Kevin)</a></li>
```

Comment:
We should probably put team roles (Product Owner, Scrum Master for Sprint 2 etc).